gcc/testsuite/rust/compile/extern_mod4.rs

```diff
@@ -0,0 +1,17 @@
+/* { dg-output "12" } */
+mod modules;
+
+extern "C" {
+    fn printf(s: *const i8, ...);
+}
+
+fn main() {
+    unsafe {
+        let fmt_s = "%d\n\0";
+        let fmt_p = fmt_s as *const str;
+        let fmt_i8 = fmt_p as *const i8;
+
+        printf(fmt_i8, modules::return_12());
+    }
+}
+>>>>>>> 1ca3f3915e7 (mod: Add two test cases for external modules)
```

Comment:
looks like a conflict marker has not been removed here?

Reply:
Oops, thanks! Yeah the PR is still very much WIP as the test cases do not pass yet